.github/workflows/build.yml

```diff
@@ -43,8 +43,40 @@ jobs:
         run: rustup target add ${{ matrix.target }}
       - name: Build
         run: cargo build --verbose --release --target ${{ matrix.target }}
+      - if: ${{ matrix.target == 'x86_64-unknown-linux-gnu' }}
+        name: Downloading dummy service for packaging
+        uses: actions/download-artifact@v4
+        with:
+          name: dummy-GalaxyCommunication.exe
+          path: $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists
+      - if: ${{ matrix.target == 'x86_64-unknown-linux-gnu' }}
+        name: Unpacking dummy service
+        run: | 
+          unzip -j $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists/dummy-GalaxyCommunications.exe.zip -d $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists
+          rm $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists/dummy-GalaxyCommunications.exe.zip
+          rm $GITHUB_WORKSPACE/steamdeck/dummy-service-folder-structure/pfx/drive_c/ProgramData/GOG.com/redists/GalaxyCommunications_here
+        
```

Comment:
I think download-artifact already does the unzip part

Reply:
Done, you were right. No need for $GITHUB_WORKSPACE either.